apps/useradmin/src/useradmin/views.py

```diff
@@ -26,7 +26,7 @@
 
 LOG = logging.getLogger(__name__)
 
-from axes.decorators import FAILURE_LIMIT, LOCK_OUT_AT_FAILURE
+from axes.conf import settings
```

Comment:
Will brake Django 1?

Reply:
As I said that django-axes==4.5.4 will support the python2.7 so I think I need to upgrade this to in ext-py file also because Django 1.11 is deprecated in 5.3.4.
So now I need to push for python2.7  in a separate branch yes or no? @romainr

Reply:
I see, yes, could you do the ext-py upgrade in this PR? (then I can rebuild the CI images with it) (we need both the upgrade + the new axes imports)

Reply:
Done https://github.com/cloudera/hue/pull/1688/commits/ddc9e896bf990ce73ab231180c7ce053ccccb72c @romainr

Reply:
One more thing that I need to add django-ipware==2.1.0 in ext-py folder also now because there is no folder for this name?

Reply:
:+1: 

And don't forget to `git add` it ;)